### PR TITLE
Translate carbon.md, carbon-intensity.md and index.md

### PR DESCRIPTION
**Pull request by ivantxu**

### CODE REVIEW

File: src/es/carbon-intensity.md [hunk 1]
```diff
@@ -1,63 +1,61 @@
...
+> [Estudios](https://ieeexplore.ieee.org/document/6128960) han mostrado que estas acciones pueden resultar en una reducción de emisiones de carbono entre el 45% y 99% dependiendo del número de renovables en la red.
 
-If you can be flexible with when and where you run workloads then you can then choose to consume electricity when the carbon intensity is less and pause when carbon intensity is high. For example, training a machine learning model at a different time or region where the carbon intensity is much lower.
+Mire tu aplicación de extremo a extremo, identifique oportunidades para ser flexible respecto a las cargas de trabajo y use la intensidad de carbono de la electricidad como señal para cuando ejecutar esas cargas de trabajo.
```
Comment (alanverdugo) [inline src/es/carbon-intensity.md]: I suppose here is should be either "Mire su aplicación" or "Mira tu aplicación".
```suggestion
Mire su aplicación de extremo a extremo, identifique oportunidades para ser flexible respecto a las cargas de trabajo y use la intensidad de carbono de la electricidad como señal para cuando ejecutar esas cargas de trabajo.
```

File: src/es/carbon-intensity.md [hunk 2]
```diff
@@ -1,63 +1,61 @@
...
 
-If your computer is plugged directly into a wind farm, then the electricity it consumes would have a carbon intensity of 0 gCO<sub>2</sub>eq/kWh since a wind farm emits no carbon to produce that electricity. Most people can’t plug directly into wind farms, they instead plug into power grids that are usually supplied with electricity from a mix of sources that produce varying amounts of carbon. Therefore, when plugged into a grid your carbon intensity is usually a number greater than 0. 
+Si su computadora está enchufada directamente a un parque eólico, entonces la electricidad que consume tendría una intensidad de carbono de 0 gCO<sub>2</sub>eq/kWh ya que un parque eólico no emite carbono para producir electricidad. La mayoría de gente no se pueden conectar directamente a un parque eólico, en su lugar se conectan a redes eléctricas que se suelen proveer de electricidad fe una mezcla de fuentes que producen distintas cantidades de carbono. Por lo tanto, cuando se conecta a una red su intensidad de carbono es normalmente un número mayor que 0.
```
Comment (alanverdugo) [inline src/es/carbon-intensity.md]: A minor typo here "se suelen proveer de electricidad **fe** una mezcla de fuentes" should be "se suelen proveer de electricidad **de** una mezcla de fuentes"

```suggestion
Si su computadora está enchufada directamente a un parque eólico, entonces la electricidad que consume tendría una intensidad de carbono de 0 gCO<sub>2</sub>eq/kWh ya que un parque eólico no emite carbono para producir electricidad. La mayoría de gente no se pueden conectar directamente a un parque eólico, en su lugar se conectan a redes eléctricas que se suelen proveer de electricidad de una mezcla de fuentes que producen distintas cantidades de carbono. Por lo tanto, cuando se conecta a una red su intensidad de carbono es normalmente un número mayor que 0.
```

File: src/es/carbon-intensity.md [hunk 3]
```diff
@@ -1,63 +1,61 @@
...
+![alt_text](/assets/images/principles/carbon-intensity-2.png "Las fuentes de energía de origen fósil habitualmente son las primeras que disminuyen su producción y las renovables las últimas.")
 
-![alt_text](/assets/images/principles/carbon-intensity-2.png "Fossil Fuel sources of power are usually scaled back first and renewables scaled back last.")
+Como subproducto de la manera en la que los mercados energéticos funcionan, a medida que la demanda de electricidad disminuye, habitualmente las fuentes basadas en combustibles fósiles de grandes emisiones reducen su producción primero y las renovables la reducen de últimas.
 
-As a by-product of the way energy markets work as demand for electricity goes down usually the high emitting fossil fuel sources of power are scaled back first with renewables scaled back last.
+> Reducir la cantidad de electricidad cpnsumida por tu aplicación puede ayudar a **disminuir** la intensidad de carbono en la mezcla de las redes eléctricas locales.
```
Comment (alanverdugo) [inline src/es/carbon-intensity.md]: Here's another minor typo. "cpnsumida" should be "cpnsumida".
```suggestion
> Reducir la cantidad de electricidad consumida por tu aplicación puede ayudar a **disminuir** la intensidad de carbono en la mezcla de las redes eléctricas locales.
```

**Review by alanverdugo — COMMENTED** [review]
Hi @ivantxu ! Your translation looks great! :)

I found a few typos, and I made the suggestions to correct them. Once that is done, we can merge this and create another PR to the main repository.

Thanks!

**Comment by ivantxu** [comment]
Cool, I'll fix them tomorrow 👍

**Comment by ivantxu** [comment]
Changes accepted, thanks for the review Alan!

**Comment by alanverdugo** [comment]
Excellent! Thanks @ivantxu ! :)
I'll merge this and then create a new PR for the main repository.

**Pull request merged**